libobs/obs-windows.c

```diff
@@ -137,10 +149,13 @@ static void log_windows_version(void)
 	bool b64 = is_64_bit_windows();
 	const char *windows_bitness = b64 ? "64" : "32";
 
+	bool arm64 = is_arm64_windows();
+	const char *arm64_windows = arm64 ? "ARM " : "";
```

Comment:
@gxalpha  This is what I talked about, we should declare whether we are ARM64 or not here.